docs/cri/config.md

```diff
@@ -237,6 +237,11 @@ version = 2
 
   # 'plugins."io.containerd.grpc.v1.cri".registry' contains config related to the registry
   [plugins."io.containerd.grpc.v1.cri".registry]
+    # Specifies a directory to look for registry configs in.
+    # Dir can be used just like /etc/docker/certs.d OR can contain a hosts.toml with more specific configurations.
+    #
+    # NOTE: Specifying this will cause the cri plugin to ignore any other registry configs specified in this configuration file.
+    hosts_dir = "/etc/containerd/cri/hosts"
```

Comment:
Can we change the path to /etc/containerd/certs.d ?

Non-CRI apps such as nerdctl will want to use the same path

Reply:
Also, that should be in the default config

Reply:
I think the reason we aren't making it default is because it causes us to ignore other configs.

Reply:
Updated the comment to use `/etc/containerd/certs.d`